mappings/net/minecraft/client/network/ClientPlayerEntity.mapping

```diff
@@ -48,6 +48,7 @@ CLASS net/minecraft/class_746 net/minecraft/client/network/ClientPlayerEntity
 	METHOD method_22419 showsDeathScreen ()Z
 	METHOD method_22420 setShowsDeathScreen (Z)V
 		ARG 1 shouldShow
+	METHOD method_26269 getMoodPercentage ()F
```

Comment:
Maybe clarify that this is for sounds?

Reply:
See [slicelime's intro vid for 20w12a](https://youtu.be/GowKC1iYz4c?t=570). On the debug screen, it says `Mood: 8%`. Should I clarify in javadocs instead?

Reply:
Added a piece of jd.